### PR TITLE
Add Test-ArubaSWPing cmdlet

### PR DESCRIPTION
**Pull request by alagoutte**
for make a ping from switch

### CODE REVIEW

File: PowerArubaSW/Public/Ping.ps1 [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        $dest = New-Object -TypeName PSObject
+
+        if ($PsBoundParameters.ContainsKey('ipv4_address')) {
+
+            $ipv4 = New-Object -TypeName PSObject
+            $ipv4 | Add-Member -name "version" -MemberType NoteProperty -Value "IAV_IP_V4"
+
+            $ipv4 | Add-Member -name "octets" -MemberType NoteProperty -Value $ipv4_address
+
+            $dest | Add-Member -name "ip_address" -membertype NoteProperty -Value $ipv4
```
Comment (CedricMoreau) [inline PowerArubaSW/Public/Ping.ps1]: membertype typo

File: PowerArubaSW/Public/Ping.ps1 [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        } elseif ($PsBoundParameters.ContainsKey('hostname')) {
+            $dest | Add-Member -name "hostname" -membertype NoteProperty -Value $hostname
```
Comment (CedricMoreau) [inline PowerArubaSW/Public/Ping.ps1]: membertype typo

File: PowerArubaSW/Public/Ping.ps1 [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        }
+        else {
+            throw "You need to use a parameter (-ipv4_address, -hostname)"
+        }
+
+        $ping = New-Object -TypeName PSObject
+        $ping | Add-Member -name "destination" -membertype NoteProperty -Value $dest
```
Comment (CedricMoreau) [inline PowerArubaSW/Public/Ping.ps1]: same as before

File: Tests/integration/Ping.Tests.ps1 [hunk 4]
```diff
@@ -0,0 +1,55 @@
+﻿#
+# Copyright 2018, Alexis La Goutte <alexis dot lagoutte at gmail dot com>
```
Comment (CedricMoreau) [inline Tests/integration/Ping.Tests.ps1]: Wrong year

File: Tests/integration/Ping.Tests.ps1 [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+        #Configure DNS
+        Set-ArubaSWDns -mode Manual -server1 $pester_dns1 -server2 $pester_dns2
+        $ping = Test-ArubaSWPing -hostname www.arubanetworks.com
+        $ping.result | Should -Be "PR_OK"
+        $ping.rtt_in_milliseconds | Should -Not -BeNullOrEmpty
+    }
+
+    AfterAll {
+        #Always remove DNS Settings...
+        Remove-ArubaSWDns -noconfirm
```
Comment (CedricMoreau) [inline Tests/integration/Ping.Tests.ps1:49]: What about previous DNS servers configured before the test ? Don't they need to be re-added ?

Comment (alagoutte) [reply Tests/integration/Ping.Tests.ps1:49]: yes... but it is not already the case on DNS test...

**Pull request merged**